wsi/drm/drmdisplaymanager.cpp

```diff
@@ -454,6 +454,10 @@ void DrmDisplayManager::setDrmMaster(bool must_set) {
   uint8_t retry_times = 0;
   do {
     ret = drmSetMaster(fd_);
+
+    // Revert me, A workaround for supporting kernel before 4.19.31
+    if (errno == EACCES)
```

Comment:
is there any drm API to check if current client is already drm master?

Reply:
No, no API is defined in xf86drm.h for the Master status. Only 2 methods are related with Master.
extern int drmSetMaster(int fd);
extern int drmDropMaster(int fd);